scripts/lint-linkcheck.ts

```diff
@@ -187,16 +217,24 @@ class BrokenLinkChecker {
 		console.log();
 	}
 
-	outputSourceFileAnnotations (brokenLinks) {
-		const annotations = [];
+	outputSourceFileAnnotations (brokenLinks: BrokenLink[]) {
+		const annotations: {
+			message: string;
+			location: {
+				file: string;
+				startLine: number;
+				startColumn: number;
+				endColumn: number;
+			}
```

Comment:
Looks like we can use `@actions/core`’s [exported type](https://github.com/actions/toolkit/blob/c4ae214c261c0d684c6bcfe6632b773b82482bad/packages/core/src/core.ts#L40-L71) for this:

```suggestion
			location: AnnotationProperties;
```

(See next suggestion for adding the import.)